src/harness/testcases/WINNF_FT_S_FDB_testcase.py

```diff
@@ -51,993 +34,148 @@ def tearDown(self):
-        'userIds': [device_c['userId']],
-        'registrationRequests': [device_c],
-        'grantRequests': [grant_0],
-        'expectedResponseCodes': [0],
-        'expectedGrantResponseCodes': [0],
-        'expectedHBResponseCodes':[500],
-        'fss_db':fss_db,
-        'gwbl_db':gwbl_db
-    }
-    writeConfig(filename, config)
-
-
-  def generate_FDB_6_default_config(self, filename):
-    '''Generates the WinnForum configuration for FDB.6'''
-    # Load device info
-    device_c = json.load(
-        open(os.path.join('testcases', 'testdata', 'device_c.json')))
-    # Send grant request
-    grant_0 = json.load(
-      open(os.path.join('testcases', 'testdata', 'grant_0.json')))
-    fss_db = json.load(open(os.path.join('testcases', 'testdata', 'fss_db.json')))
-    # Load GWBL database which includes at least one FSS site near the above registered CBSD location
-    gwbl_db = json.load(open(os.path.join('testcases', 'testdata', 'gwbl_db.json')))
-
-    # Create the actual config.
-    config = {
-        'fccIds': [(device_c['fccId'], 47)],
-        'userIds': [device_c['userId']],
-        'registrationRequests': [device_c],
-        'grantRequests': [grant_0],
-        'expectedResponseCodes': [0],
-        'expectedGrantResponseCodes': [0,400],
-        'expectedHBResponseCodes':[0],
-        'expectedRelinquishResponseCodes': [0],
-        'fss_db':fss_db,
-        'gwbl_db':gwbl_db
-    }
-    writeConfig(filename, config)
-
-
-  def generate_FDB_7_default_config(self, filename):
-    '''Generates the WinnForum configuration for FDB.7'''
-    # Load device info
-    device_c = json.load(
-        open(os.path.join('testcases', 'testdata', 'device_c.json')))
-    fcc_id_db = json.load(open(os.path.join('testcases', 'testdata', 'fcc_id_db.json')))
-    # Create the actual config.
-    config = {
-        'fccIds': [(device_c['fccId'], 47)],
-        'userIds': [device_c['userId']],
-        'registrationRequests': [device_c],
-        'expectedResponseCodes': [0,103],
-        'fcc_id_db':fcc_id_db
-
-    }
-    writeConfig(filename, config)
-
-
-  def generate_FDB_8_default_config(self, filename):
-    '''Generates the WinnForum configuration for FDB.8'''
-    # Load device info
-    device_c = json.load(
-        open(os.path.join('testcases', 'testdata', 'device_c.json')))
-    # Send grant request
-    grant_0 = json.load(
-      open(os.path.join('testcases', 'testdata', 'grant_0.json')))
-    fss_db = json.load(open(os.path.join('testcases', 'testdata', 'fss_db.json')))
-
-    # Create the actual config.
-    config = {
-        'fccIds': [(device_c['fccId'], 47)],
-        'userIds': [device_c['userId']],
-        'registrationRequests': [device_c],
-        'grantRequests': [grant_0],
-        'expectedResponseCodes': [0],
-        'expectedGrantResponseCodes': [0],
-        'expectedHBResponseCodes':[500],
-        'fss_db':fss_db
-    }
+        'registrationRequest': [device_a],
```

Comment:
We need to support multiple registrations and grants. Please double-check the test spec.

Reply:
Added CatB device   and sending multiple registration and grants